dataactcore/models/domainModels.py

```diff
@@ -25,4 +25,14 @@ class ObjectClass(Base):
     __tablename__ = "object_class"
     object_class_id = Column(Integer, primary_key=True)
     object_class_code = Column(Text,nullable=False,index=True,unique=True)
-    object_class_name = Column(Text)
\ No newline at end of file
+    object_class_name = Column(Text)
+
+class ProgramActivity(Base):
+    __tablename__ = "program_activity"
+    program_activity_id = Column(Integer, primary_key=True)
+    budget_year = Column(Text,nullable=False)
```

Comment:
We are doing comparisons between this and beginningperiodofavailability in files B and C, which we are representing as text.  Since we're filtering by values from that, I'd prefer that they be the same type to avoid complications.